### DISCUSSION TITLE
Good Enough for me; Can I use it now?

### DISCUSSION THREAD

**Discussion by FarYetNear** [microsoft/winget-cli#677] [General] [answered]
Is there a way to grab the existing winget tool and put it in use without each machine/user being in the Insider Program?   The tool is good enough for what I need to do in a 1200 device environment, and I'd like to get going now.

**Comment by Shatungoo** [comment, marked as answer]
You can install winget manually(i did the same on my company notebook). Latest release can be found here https://github.com/microsoft/winget-cli/releases. 
Just note- currently you will need to update winget manually in this case.
